workers_tail.go

```diff
@@ -0,0 +1,114 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+var (
+	ErrMissingScriptName = errors.New("required script name missing")
+	ErrMissingTailID     = errors.New("required tail id missing")
+)
+
+type WorkersTail struct {
+	ID        string     `json:"id"`
+	URL       string     `json:"url"`
+	ExpiresAt *time.Time `json:"expires_at"`
+}
+
+type StartWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+}
+
+type StartWorkersTailResponse struct {
+	Response
+	Result WorkersTail
+}
+
+type ListWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+}
+
+type ListWorkersTailResponse struct {
+	Response
+	Result WorkersTail
+}
+type DeleteWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+	TailID     string
+}
+
+// StartWorkersTail Starts a tail that receives logs and exception from a Worker
+//
+// API reference: https://api.cloudflare.com/#worker-tail-logs-start-tail
+func (api *API) StartWorkersTail(ctx context.Context, params StartWorkersTailParameters) (WorkersTail, error) {
```

Comment:
the method signature conventions have moved ever so slightly since your last PR 😄 we now have a new concept of a `ResourceContainer` that helps denote the account/zone details as a part of the method while keeping parameters separate - see https://github.com/cloudflare/cloudflare-go/blob/master/docs/experimental.md#consistent-crud-method-signatures. example usage: https://github.com/cloudflare/cloudflare-go/blob/27fcb0506f52661d7e161178c521c98193c106a4/tunnel.go#L218

Reply:
cleaned up in 47ab54b if you're interested.

Reply:
Considering I have another PR on the way, this is good to know.